txscript/data/script_tests.json

```diff
@@ -1493,6 +1493,28 @@
     "BIP66 example 4, with DERSIG, non-null DER-compliant signature"
 ],
 [
+    "0",
+    "0x21 0x038282263212c609d9ea2a6e3e172de238d8c39cabd5ac1ca10646e23fd5f51508 CHECKSIG NOT",
+    "DERSIG,NULLFAIL",
+    "OK",
+    "BIP66 example 4, with DERSIG and NULLFAIL"
+],
+[
+    "0x09 0x300602010102010101",
+    "0x21 0x038282263212c609d9ea2a6e3e172de238d8c39cabd5ac1ca10646e23fd5f51508 CHECKSIG NOT",
+    "DERSIG,NULLFAIL",
+    "NULLFAIL",
+    "BIP66 example 4, with DERSIG and NULLFAIL, non-null DER-compliant signature"
+],
+[
+
+    "0x09 0x300602010102010101",
```

Comment:
This entry is a repeat of the entry just before the latest two entries you added.  In other words, it appears the two previous entries should be inserted before it and this one removed.